freemindlatex/example/slides.mm

```diff
@@ -0,0 +1,19 @@
+<map version="1.0.1">
```

Comment:
This file is currently identical to the report.mm. For report, we should try a different content, e.g. to generate a doc with two sections.